src/plugins/ajs-destination/loader.ts

```diff
@@ -42,6 +39,7 @@ export async function loadIntegration(
 ): Promise<LegacyIntegration> {
   const pathName = normalizeName(name)
   const obfuscatedPathName = obfuscatePathName(pathName, obfuscate)
+  const path = getNextIntegrationsURL()
```

Comment:
calling this method outside of the global scope make them easier to reason about, especially since it's a wrapper around a global, we don't need to worry about stale references.